src/components/RecurringPaymentsTableRow.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import moment from 'moment';
+
+import { TableRow, TableCell } from '@material-ui/core';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+// import { CheckCircleOutlineIcon } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+import theme from '../theme'; // Double-check this; styling should be done on top level?
```

Comment:
I couldn't get top level imports to work correctly.  Leave it like this for now.